frontend/src/config/i18n/en/layers.json

```
@@ -0,0 +1,23 @@
+{
+    "header": "Layers",
+    "base": "Base Layer",
+    "plants": "Plants Layer",

```

Please also add other translations

```suggestion
    "header": "Layers",
    "base": "Base",
    "plants": "Plants",

```